client/src/components/FilterServicesAndFeatures/FilterServicesAndFeatures.tsx

```diff
@@ -49,64 +53,52 @@ const FilterServicesAndFeatures: React.FC<Props> = (props: Props) => {
     <>
       {!props.loading ? (
         <>
-          <div className='filters' style={{ margin: '10px' }}>
-            <div className='form-group'>
-              <label htmlFor='features'>Features</label>
+          <div className='Filters-Container'>
+            <div className='Filters-FormGroup'>
+              <label htmlFor='features' className='Filters-Label_header'>
+                Features
+              </label>
 
               {props.featuresArr?.map((feature, id, index) => (
-                <div
-                  className='form-check'
-                  style={{ textTransform: 'capitalize' }}
-                  key={`${feature}_` + index}
-                >
-                  <input
-                    className='form-check-input'
-                    type='checkbox'
-                    name={`feature-${feature[0]}`}
-                    id={feature[1]}
-                    defaultChecked={feature[2].isChecked}
-                    value={id}
-                    onChange={onFeatChange}
-                  />
-                  <label className='form-check-label' htmlFor={feature[1]}>
+                <div className='Filters-FormCheck' key={`${feature}_` + index}>
+                  <input className='Filters-FormCheckInput' type='checkbox' name={`feature-${feature[0]}`} id={feature[1]} defaultChecked={feature[2].isChecked} value={id} onChange={onFeatChange} />
+                  <label className='Filters-FormCheckLabel' htmlFor={feature[1]}>
```

Comment:
This looks great overall. I know figma docs don't show the reset filter button, so will style that later, as well as adding text alignment.

Reply:
Thank you @cynthiaeddy! I have updated the layout to include the button styles.